spotdl/search/songObj.py

```diff
@@ -63,9 +64,16 @@ def from_url(cls, spotifyURL: str):
             duration
         )
 
+        # try to get lyrics from Genius 
```

Comment:
```suggestion
        # try to get lyrics from Genius
```

![image](https://user-images.githubusercontent.com/42355410/110792437-b1c10900-8273-11eb-84ad-5de79c5ebcf0.png)